<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Search:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>45</y>
     <width>491</width>
     <height>225</height>
    </rect>
   </property>
   <property name="title">
    <string>Patient List</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>By Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>By Date</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>By Doctor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>By Phone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>By Status</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
